src/main/java/org/folio/inventory/dataimport/handlers/actions/HoldingsUpdateDelegate.java

```diff
@@ -78,7 +79,14 @@ private JsonObject retrieveParsedContent(ParsedRecord parsedRecord) {
   private Future<HoldingsRecord> getHoldingsRecordById(String holdingsId,
                                                        HoldingsRecordCollection holdingsRecordCollection) {
     Promise<HoldingsRecord> promise = Promise.promise();
-    holdingsRecordCollection.findById(holdingsId, success -> promise.complete(success.getResult()),
+    holdingsRecordCollection.findById(holdingsId, success -> {
+        if (success.getResult() == null) {
+          LOGGER.error(format("Can't find Holdings by id: %s ", holdingsId));
```

Comment:
@VRohach for loggers we can rewrite like this: LOGGER.error("Can't find Holdings by id: {} ", holdingsId));

Reply:
@SerhiiNosko Changed.